src/sign/storage.tf

```diff
@@ -0,0 +1,16 @@
+module "io_sign_storage" {
```

Comment:
this storage needs to be public o private?

Reply:
public

Reply:
perfect this storage it's ok :)